Comprehensive EDGAR overview and usage quickstart guide (#313)
sec-edgar/sec-edgar: docs/source/about.rst

3. **Types of Filings**: EDGAR contains registration statements, periodic reports, and other forms filed by companies. Common filings include:
   - 10-K (Annual reports)
   - 10-Q (Quarterly reports)
   - 8-K (Current reports)
   - S-1 (Initial public offering registration statements)
   - 13F (Institutional investment manager holdings reports)
   - And many more

4. **Historical Data**: The system provides access to filings dating back to 1994, offering a vast repository of historical financial data.

## How secedgar Helps

secedgar streamlines the process of interacting with the EDGAR database by providing:

1. **Batch Downloads**: Retrieve filings from multiple companies in a single operation.
2. **Customizable Queries**: Specify date ranges, filing types, and other parameters to get exactly the data you need.
3. **Automated Parsing**: secedgar can parse the retrieved filings, making it easier to extract specific information.
4. **Programmable Interface**: Integrate SEC filing retrieval into your Python scripts and applications.

## Getting Started with secedgar

> To begin using secedgar, you'll typically follow these steps:

Reviewer: ```suggestion
To begin using secedgar, you can follow these steps:
```

1. Install the package: `pip install secedgar`
2. Import the necessary modules in your Python script

> 3. Create a `CompanyFilings` or `FilingType` object

Reviewer: ```suggestion
3. Create a `CompanyFilings` and `FilingType` object
```
